deployments/pulse3d/services/pulse3d/src/main.py

```diff
@@ -600,8 +612,9 @@ def _format_tuple_param(
         default_values = (default_values,) * len(options)
 
     # set any unspecified values to the default value
+    # pulse3d does not like float values
```

Comment:
We could also have pulse3d enforce these values being int if needed, make it the responsibility of the code using the value instead of the caller

Reply:
Yeah I'll make a note to also add it to pulse3d